model/simulationtests/output_objects.py

```diff
@@ -88,5 +88,13 @@
 # output_table.getSummaryReport(1).get == "AdaptiveComfortSummary"
 # output_table.summaryReportIndex("AdaptiveComfortSummary").get == 1
 
+###############################################################################
+#                       OUTPUTCONTROL:RESILIENCESUMMARIES                     #
+###############################################################################
+if openstudio.VersionString(openstudio.openStudioVersion()) > openstudio.VersionString("3.9.0"):
```

Comment:
This is added to an _existing_ test, so we need the version comparison?

Reply:
yes. I don't have to have 3 new tests just for this dumb object, so added to an existing one.